cypress/integration/form.ts

```diff
@@ -1,70 +1,65 @@
+import { AuthRoles } from '../support/commands';
+
 describe('Test Form', () => {
-  beforeEach(() => {
-    cy.getCookies().should('be.empty');
-    cy.setCookie('hackneyToken', Cypress.env('TEST_KEY_ADMIN_DEV'));
-    cy.getCookie('hackneyToken').should(
-      'have.property',
-      'value',
-      Cypress.env('TEST_KEY_ADMIN_DEV')
-    );
-    cy.visit(`${Cypress.env('HOST')}/form/test/first-step`);
-  });
+  describe('As a user in the Admin Dev group', () => {
+    it('submit the form and show a confirmation page', () => {
+      cy.visitAs('/form/test/first-step', AuthRoles.AdminDevGroup);
 
-  it('Should work properly', () => {
-    cy.intercept(
-      {
-        method: 'POST',
-        url: '/test',
-      },
-      []
-    ).as('apiCheck');
+      cy.intercept(
```

Comment:
What does `intercept` do? Is it stubbing the POST request?

Reply:
That's absolutely right, it is a bit like a mock at the HTTP level